examples/kubevirt-disk-uploader-tekton.yaml

```diff
@@ -212,17 +217,20 @@ metadata:
   name: kubevirt-disk-uploader-pipeline
 spec:
   params:
+  - name: EXPORT_SOURCE_KIND
+    description: "Kind of the export source"
+    type: string
   - name: POD_NAME
-    description: "Name of the virtual machine"
+    description: "Name of the export source"
```

Comment:
this description does not match to the name of the param

Reply:
I'd name it as `EXPORT_SOURCE_NAME` but the `POD_NAME` is required and hardcoded in `ownerreference.go`.

Reply:
You're right, did it by mistake of copy-paste.